src/enums/enums.js

```diff
@@ -14,5 +14,16 @@ export const COLOR = Object.freeze({
  */
 export const DEFAULT_TEAM = Object.freeze({
     TRI_CODE: 'TOR',
+    URL_NAME: 'raptors',
     TEAM_ID: 1610612761,
 });
+
+/**
+ * Enum for philadelphia 76's team, whose team url name needs to be changed to 'sixers'
+ * @readonly
+ * @enum {string}
+ */
+export const SIXERS_TEAM = Object.freeze({
+    URL_NAME: 'sixers',
+    TEAM_ID: 1610612755,
+});
```

Comment:
This works, but it can be improved. So this file should look like so:

```
/**
 * Enum for common colors.
 * @readonly
 * @enum {string}
 */
export const COLORS = Object.freeze({
    GREY: '#F5F5F5',
});

/**
 * Enum for teams object.
 * @readonly
 * @enum {{ TRI_CODE: string, NAME: string, ID: string }}
 */
export const TEAMS = Object.freeze({
    TOR: {
      TRI_CODE: 'TOR',
      NAME: 'raptors',
      ID: '1610612761',
    },
    PHI: {
      TRI_CODE: 'PHI',
      NAME: 'sixers',
      ID: '1610612755',
    },
});
```

Notice that I changed `COLOR` to `COLORS`
Moreso, now when you need to use these enums, for example if you want to use these values you call them like so:

```
import { TEAMS } from '../../enums/enums'

...

TEAMS.TOR.TRI_CODE //  'TOR'
TEAMS.TOR.ID //  '1610612755'
TEAMS.PHI.NAME // 'sixers'

```

Reply:
Also let's rename this file from `enums.js` to just `index.js` to follow your current file naming convention